Makefile

```diff
@@ -6,7 +6,7 @@ image:
 	docker build --rm -t $(IMAGE_NAME) .
 
 test: image
-	docker run --tty --interactive --rm $(IMAGE_NAME) bundle exec rake
+	docker run --tty --interactive --rm $(IMAGE_NAME) sh -c "cd /usr/src/app && bundle exec rake"
```

Comment:
I'm curious why this was needed, were the tests not running before?

You can use

```
docker run --tty --interactive --rm --workdir /usr/src/app $(IMAGE_NAME) bundle exec rake
```

Instead of jumping to `sh` so you can `cd` as part of the command.